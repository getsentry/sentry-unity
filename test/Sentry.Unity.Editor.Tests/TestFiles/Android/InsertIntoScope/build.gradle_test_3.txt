allprojects {
    repositories {
        google()
        jcenter()
        flatDir {
            dirs "${project(':unityLibrary').projectDir}/libs"
        }
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
