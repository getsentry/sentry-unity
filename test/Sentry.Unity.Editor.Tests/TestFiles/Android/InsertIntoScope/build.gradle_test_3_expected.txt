allprojects {
    repositories {
        maven { url "${project(':unityLibrary').projectDir}/android-sdk-repository" }
        google()
        jcenter()
        flatDir {
            dirs "${project(':unityLibrary').projectDir}/libs"
        }
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
